### PR TITLE
Ratelimit save command progress bar updates

### PR DESCRIPTION
**Pull request by 132ikl**
# Description

Currently, the `save -p` command updates the progress animation each time any data is written. This PR rate limits the animation so it doesn't play as fast.

Here's an asciinema of [current behavior](https://asciinema.org/a/8RWrWTozQSceqx6tYY7kzblqj) and [proposed behavior](https://asciinema.org/a/E1pi0gMwMwFcxVHOy9Fv1Kk6R).

# User-Facing Changes

* `save -p` progress bar has a smoother animation

# Tests + Formatting

- :green_circle: `toolkit fmt`
- :green_circle: `toolkit clippy`
- :green_circle: `toolkit test`
- :green_circle: `toolkit test stdlib`

# After Submitting
N/A

### CODE REVIEW

File: crates/nu-command/src/filesystem/save.rs [hunk 1]
```diff
@@ -499,7 +500,10 @@ fn stream_to_file(
                     let len = buf.len();
                     reader.consume(len);
                     bytes_processed += len as u64;
-                    bar.update_bar(bytes_processed);
+                    if last_update.elapsed() >= Duration::from_millis(75) {
```
Comment (fdncred) [inline crates/nu-command/src/filesystem/save.rs:503]: Just curious, why 75?

Comment (132ikl) [reply crates/nu-command/src/filesystem/save.rs:503]: Based purely on what looked good to me. I thought 50ms was too fast and 100ms was a bit too slow.

**Review by IanManske — APPROVED** [review]
Looks good, thanks for finishing this TODO! (I think any interval around 75-100ms should be fine.)

**Comment by fdncred** [comment]
Thanks

**Pull request merged**